components/speedreader/common/speedreader_result.mojom

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+module speedreader.mojom;
+
+import "mojo/public/mojom/base/values.mojom";
```

Comment:
im not sure if this is needed. i dont see the usage of `Value` type in the file.